change/@ni-nimble-blazor-eed57bcb-a0ee-4959-bdc1-c9e233274889.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Createt Blazor Wrapper for Wafer Map Component",
+  "packageName": "@ni/nimble-blazor",
+  "email": "33986780+munteannatan@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
Make sure to update the blazor status to incubating: 

https://github.com/ni/nimble/blob/24bb081b2930a9720f8420b6a36aaecedbff75ba/packages/nimble-components/src/tests/component-status.stories.ts#L527-L535